src/test/resources/tinylog-test.properties

```diff
@@ -1,2 +1,3 @@
 level = info
 writer = console
+level@org.jabref.model.entry.BibEntry = debug
```

Comment:
Is this only meant as an example, or actually desired?

Reply:
I wanted to choose a class not having extensive debug statements. I wanted to keep a line directly working. Think of novices not sure about comment characters in `.properties` file etc.

Reply:
And we reference it in the udpated howto:

> In the `.properties` file, this is done for `org.jabref.model.entry.BibEntry`.

Reply:
Are comments in a properties file allowed? If yes, it would be nice to give a few examples (i.e. activate debug logging for a particular class, for a module, for everything).

Reply:
Hash character `#this is a comment work`